Terasology/Health
# Block Damage systems

Adds required assets and system for functioning of damage to blocks.

How to test:
1. Create a new game with `BuilderSampleGameplay` template with `Health` Module activated in advanced settings
2. Try to damage a block. See that the block takes damage before being destroyed.
